### PR TITLE
fix: Enforce kw args featureservice

### PR DESCRIPTION
**Pull request by kevjumba**
**What this PR does / why we need it**:
Enforce kw args in feature service for convergence.
**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: sdk/python/tests/unit/test_feature_service.py [hunk 1]
```diff
@@ -55,3 +57,20 @@ def test_hash():
 
     s4 = {feature_service_1, feature_service_2, feature_service_3, feature_service_4}
     assert len(s4) == 3
+
+
+def test_feature_view_kw_args_warning():
+    # source_class = request.param
```
Comment (achals) [inline sdk/python/tests/unit/test_feature_service.py:63]: Rm?

File: sdk/python/tests/unit/test_feature_service.py [hunk 2]
```diff
@@ -55,3 +57,20 @@ def test_hash():
 
     s4 = {feature_service_1, feature_service_2, feature_service_3, feature_service_4}
     assert len(s4) == 3
+
+
+def test_feature_view_kw_args_warning():
```
Comment (achals) [inline sdk/python/tests/unit/test_feature_service.py:62]: Add a test for the kwargs version not raising any deprecation warnings?

Comment (kevjumba) [reply sdk/python/tests/unit/test_feature_service.py:62]: Done

File: sdk/python/tests/unit/test_feature_service.py (hunk 1 above)
Comment (kevjumba) [reply sdk/python/tests/unit/test_feature_service.py]: Done.

**Review by achals — APPROVED** [review]
/lgtm

**Pull request merged**